src/index.js

```diff
@@ -55,6 +62,7 @@ function fastbootExpressMiddleware(distPath, options) {
       if (error.name === "UnrecognizedURLError") {
         next();
       } else {
+        next(error);
```

Comment:
This should go after the work we do to `res`, so that middleware can potentially recover